dists/install.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
+function sha256sum() {
+  shasum -a 256 "$@"
+}
+
```

Comment:
This is the actual fix

Reply:
question: from what I understood of the actual problem, the call to `sha256sum` works as this script used to work on Benjamin's Mac. It's the introduction of the function `check_command_exists` that made the install fail. Isn't it?

Reply:
Not really, it never worked on Benjamin's Mac since the addition of the call to `sha256sum`.
This binary seems not installed by default on MacOS.